client/components/MessageCenter/Topic/Topic.tsx

```diff
@@ -55,7 +66,7 @@ const Topic: React.FC<TopicProps> = (props) => {
   useEffect(() => {
     const eventHandler = (args: [message: Message]) => {
       const [message] = args
-      dispatch(MessageCenterActions.addMessage({ message, topic }))
+      if (message) dispatch(MessageCenterActions.addMessage({ message, topic }))
```

Comment:
why are we sending empty message ?

Reply:
sorry, it was incomplete. what I wanted to to was making it more generic to receive any possible change. in this case, it the status of the topic is changed, this change will be reflected to all the open topics. the use case is:

two admins or an admin and a reviewer are working on the same topic, if the status of the topic is changed the resolve button disappear for everyone.

Reply:
Sure , but then you need a new message type. You can't use the same event type